src/tr1/game/level.c

```diff
@@ -594,15 +594,9 @@ bool Level_Initialise(const GF_LEVEL *const level)
         Game_SetCurrentLevel((GF_LEVEL *)level);
     }
     GF_SetCurrentLevel((GF_LEVEL *)level);
-    g_FlipEffect = -1;
 
     Overlay_HideGameInfo();
 
-    g_FlipStatus = 0;
-    for (int32_t i = 0; i < MAX_FLIP_MAPS; i++) {
-        g_FlipMapTable[i] = 0;
-    }
-
```

Comment:
Is it safe to remove this?

Reply:
Yes, this has been merged into `Room_InitialiseFlipStatus`, which we call from `Level_ReadRooms`. So it just happens a little later, but still before the level actually finishes loading.